lib/views/notifications_view/notifications_overview.dart

```diff
@@ -0,0 +1,46 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_riverpod/flutter_riverpod.dart';
+import 'package:gocast_mobile/providers.dart';
+
+import 'notifications_screen_view.dart';
+
+class MyNotifications extends ConsumerStatefulWidget {
+  const MyNotifications({super.key});
+
+  @override
+  MyNotificationsState createState() => MyNotificationsState();
+}
+
+class MyNotificationsState extends ConsumerState<MyNotifications> {
+  @override
+  void initState() {
+    super.initState();
+    _fetchData();
+  }
+
+  void _fetchData() {
+    Future.delayed(Duration.zero, () async {
+      try {
+        await ref.read(userViewModelProvider.notifier).fetchFeatureNotifications();
+        await ref.read(userViewModelProvider.notifier).fetchBannerAlerts();
+      } catch (e) {
+        // Handle errors if needed
```

Comment:
Handle Error Here

Reply:
Done ✅ (Error is already handled in UserViewModel)